src/common/cm/cm_test.cpp

```diff
@@ -507,17 +507,12 @@ CM_BoundsIntersect
 */
 bool CM_BoundsIntersect( const vec3_t mins, const vec3_t maxs, const vec3_t mins2, const vec3_t maxs2 )
```

Comment:
I wonder if those functions should not be inline to help compiler inlining them (I know they don't necessarily obey, but still)?

Reply:
No that's not what `inline` does.

Reply:
yes, ofc they need to go in header, but my question stands still.

Reply:
The compiler already inlines them very well.